web/src/main/resources/log4j2.component.properties

```
@@ -1 +1 @@
-log4j.configurationFile=classpath:log4j2.properties,${sys:georchestra.datadir}mapstore/log4j2.properties
+log4j.configurationFile=${sys:georchestra.datadir}/mapstore/log4j2.properties
```

It is ok for me. But I didn't get the reason why you want to exclude the default file. 
```suggestion
log4j.configurationFile=classpath:log4j2.properties,${sys:georchestra.datadir}/mapstore/log4j2.properties
```